### DISCUSSION TITLE
autocomplete for #include never shows .c&.cpp files. only shows headers .h files

### DISCUSSION THREAD

**Discussion by masriomarm** [microsoft/vscode-cpptools#8891] [Q&A]
Greetings,
i have an issue as described in the title.
when trying to include a .c/.cpp file the autocomplete never suggests  source files. it shows many file types but never source files. it shows .h files, .txt files, .pdf files, and many others. but never list my .c/.cpp files.
any idea how to solve that?

**Comment by sean-mcmanus** [comment]
That is intentional, "by design" behavior with the include completion feature. You could file a feature request to add some setting to enable that...or I could "convert" this into a feature request.

**Reply by sean-mcmanus** [reply]
Ah, it's https://github.com/microsoft/vscode-cpptools/issues/8896
